src/opera/instance/topology.py

```diff
@@ -53,5 +53,12 @@ def undeploy(self, verbose, workdir, num_workers=None):
     def write(self, data, instance_id):
         self.storage.write_json(data, "instances", instance_id)
 
+    def write_all(self):
+        for node in self.nodes.values():
+            node.write()
+
     def read(self, instance_id):
         return self.storage.read_json("instances", instance_id)
+
+    def set_storage(self, storage):
+        self.storage = storage
```

Comment:
Why do you need to (re)set storage?

Reply:
Basically we have 2 scenarios. 
- we have 2 topologies with different storages, and we want to update from one to another. It is a simple one, at the end we would have all instances as `initial `in the first storage and all `deployed `in the second
- we have 2 topologies but we want to keep only one storage, and we want overwrite first instance states with second states
In case we want to overwrite we assign old storage to a new topology.

Reply:
Ok good, I see that you have that `overwrite` parameter - but why don't we expose that as a CLI option `--overwrite/-o`?

Reply:
Maybe be it is a good idea but again as a separate enhancement. Its better to discuss it first, it may be not as trivial as it seem.